code/game/machinery/machinery.dm

```diff
@@ -157,10 +157,11 @@ Class Procs:
 		PoolOrNew(/obj/effect/overlay/temp/emp, loc)
 	..()
 
-/obj/machinery/proc/open_machine()
+/obj/machinery/proc/open_machine(var/drop = 1)
```

Comment:
It's just for when you want to create a var for an atom during a proc or initialized.